atlascope/core/rest/endpoints/investigation_endpoints.py

```diff
@@ -4,7 +4,7 @@
 from rest_framework.response import Response
 from rest_framework.viewsets import GenericViewSet
 
-from atlascope.core.models import (
+from atlascope.core.models import (  # Tour,; TourCreateSerializer,; TourSerializer,
```

Comment:
Can we get rid of this comment, or does it have some effect on the tooling or something?

Reply:
removed in 19ba0032d9cea5ca607ae52bea612feec0d07ebd